### ISSUE TITLE
~leave_here alias for 'not is_included'

### ISSUE THREAD

**Issue by majidaldo** [ESSS/conda-devenv#120]
I rely heavily on ` not is_included` to process my devenvs. I'm assuming that this is the way it's expressed in use cases (to be useful). But thinking about the negation of inclusion is too much to think about. I find myself always having to think about it. I think it's just simpler to think about the intent of its use.

**Comment by prusse-martin** [comment]
Yes.
Our brains have problems to process negatives.

`leave_here`...
The first thing that come to my mind is an `is_root` but I think it is much similar to `root`(anoter var already present).
@nicoddemus  do you have any suggestion about a variable name?

As a temporary workaround you could add (maybe at the top) `{% set leave_here = not is_included %}` and then use `leave_here` as a variable normally.

**Comment by nicoddemus** [comment]
> @nicoddemus do you have any suggestion about a variable name?

Not at the moment :/

**Comment by majidaldo** [comment]
> Yes.
> Our brains have problems to process negatives.
> 
> `leave_here`...
> The first thing that come to my mind is an `is_root` but I think it is much similar to `root`(anoter var already present).
> @nicoddemus do you have any suggestion about a variable name?
> 
> As a temporary workaround you could add (maybe at the top) `{% set leave_here = not is_included %}` and then use `leave_here` as a variable normally.

Yes. I already do something like that https://github.com/ESSS/conda-devenv/issues/112#issuecomment-672290972

**Comment by majidaldo** [comment]
Well, `root` should be ~`root_dir`.

To use 'root' language, `at_root`, `inclusion_root`, `at_inclusion_root`? The user would have to understand the processing of the devenvs as a tree where each devenv is a 'root' for included devenvs (not saying it's a bad thing).

Yeah there are two 'tree' concepts to deal with: the filesystem and the inclusion tree.

**Comment by prusse-martin** [comment]
My bad, you are right `root` is relative to the current devenv file.

Maybe we could deprecate the `root` variable naming and name it `file_location` (or `devenv_file_location`) since there is a request to support referencing devenv files from the web and use the alias `is_top_level = not is_included` to avoid root related terms.

**Comment by majidaldo** [comment]
> My bad, you are right `root` is relative to the current devenv file.
> 
> Maybe we could deprecate the `root` variable naming and name it `file_location` (or `devenv_file_location`) since there is a request to support referencing devenv files from the web and use the alias `is_top_level = not is_included` to avoid root related terms.

`this_file_path`, and `this_file_dir`. 
and
`is_base`

?
